### DISCUSSION TITLE
How to install on a Mac.  Please help.

### DISCUSSION THREAD

**Discussion by jb8484** [stashapp/stash#2362] [Q&A] [answered]
I have been using stashapp for a while on windows and recently switched to a Mac Mini M1.  I downloaded stash-macos-applesilicon from the latest assets (as well as a few older ones, and the intel version), but i have no idea what to do with the file.  It doesnt have a file extension and doesnt seem to be an app, a dmg, or a pkg installer.  what do i do with this file?

Sorry i'm such a noob, but i'm not a coder.  On windows i just downloaded stash-win.exe and ran that file.  This doesnt seem as straight forward.  I've looked through the read me and tried searching for some instructions, but haven't really found anything.  Everything seems to be about how to configure once it's installed, but I cant even get it installed.  If i've missed some installation guide, please provide a link.  Thank you.

**Comment by kermieisinthehouse** [comment, marked as answer]
It's a regular executable. A bundle version was proposed but has to resolve some bugs first.

Open up a terminal, then
```
cd Downloads
chmod +x stash-macos-applesilicon
./stash-macos-applesilicon
```

**Reply by jb8484** [reply]
Thank you!

**Reply by PotatoFish44** [reply]
Hey I'm in the exact same boat, but the above doesn't seem to work why I enter it into terminal. Can you provide more detail on what to do. 

Thanks!

**Reply by BuntaroBuntaro** [reply]
For anyone else coming across this, once the program launches in the terminal you have to go to localhost:9999 (or whatever port it tells you) as if it's a website in a web browser.

**Reply by wqqy** [reply]
Thank you!

**Reply by farresh** [reply]
thank you this worked for me too and tipped/become a contributor

**Comment by yorks-dev** [comment]
i am getting this error here - zsh: bad CPU type in executable: ./stash-macos-applesilicon

**Reply by kermieisinthehouse** [reply]
If you don't have an apple silicon / ARM device, you'll need to stash-macos-intel executable

**Reply by yorks-dev** [reply]
yeah i mistakenly tried the app on my 2015 mac instead of my m1 mac like a dumbass HAHA.
